### DISCUSSION TITLE
Where can I see a specific version of golangci-lint documentation?

### DISCUSSION THREAD

**Discussion by mateusoliveira43** [golangci/golangci-lint#4247] [Q&A] [answered]
I saw that one project I work on uses `golangci-lint` version 1.54.2, and saw there were some outdated configuration in its config file. But did not saw in the docs (https://golangci-lint.run/) where to see docs for that specific version. 

How should I proceed in a case like this?

**Comment by ldez** [comment, marked as answer]
Hello,

the best approach is to update golangci-lint.

We maintain and document only one version (the current).

**Comment by mateusoliveira43** [comment]
@ldez thanks for the quick response!

Is there any auxiliary commands to help update  golangci-lint? Like something that checks for use of deprecated linters, or unused configs, etc

**Reply by ldez** [reply]
not really, you have to run the new version, the deprecated linters will be displayed in the logs. For the configuration, you can use the [JSON schema](https://json.schemastore.org/golangci-lint.json) to validate it.

**Reply by mateusoliveira43** [reply]
Thanks!
